### PR TITLE
make settings['workdir'] always accessible; absolute imports

### PR DESCRIPTION
**Pull request by shoshber**
@oesteban @rwblair I would like confirmation that this is ok from one of you before I merge

### CODE REVIEW

**Comment by oesteban** [comment]
This is a great idea. Maybe you can also address this problem on mriqc: https://github.com/poldracklab/mriqc/issues/111

I'm pretty sure this will also happen in the preprocessing pipeline.

**Comment by rwblair** [comment]
I think it would be best to fail with an error message and force the user to choose an explicit work directory. The output to the work directory can take up a non trivial amount of space so it would be good to make the user think about where its all going to end up...

But a warning message saying that no work directory was provided and that the current directory is being used may be enough. If something bad happens to their current working directory/partition they can at least go back and see what happened.

File: fmriprep/run_workflow.py [hunk 1]
```diff
@@ -116,18 +116,19 @@ def main():
...
-        log_dir = op.join(settings['work_dir'], 'log')
-        if not op.exists(log_dir):
-            os.makedirs(log_dir)
-
-        # Set nipype config
-        ncfg.update_config({
-            'logging': {'log_directory': log_dir, 'log_to_file': True},
-            'execution': {'crashdump_dir': log_dir}
-        })
+    settings['work_dir'] = (op.abspath(opts.work_dir)
```
Comment (oesteban) [inline fmriprep/run_workflow.py]: This is too verbose, IMHO. For this very same behavior, it is enough to:
1. Define a ```default=os.getcwd()``` in the argument (line 95)
2. Do ```settings['work_dir'] = op.abspath(opts.work_dir)

Comment (shoshber) [inline fmriprep/run_workflow.py]: I see what you're saying. I get too excited about ternary operators!

That said, I'm about to re-write this anyway, to follow Ross's recommendation (output a warning, and proceed).

File: fmriprep/run_workflow.py [hunk 2]
```diff
@@ -97,7 +96,7 @@ def main():
     opts = parser.parse_args()
 
     if opts.version:
-        print('fmriprep version ' + __version__)
+        logging.info('fmriprep version ' + __version__)
```
Comment (oesteban) [inline fmriprep/run_workflow.py]: Why should we use logging for an option that only shows the current version? Isn't it a little bit of overkilling the problem?

**Comment by oesteban** [comment]
@shoshber I have added #78 for a more consistent setting of output and working directories, including locks for multiprocessing operation. I suggest to merge #78 first and then this one, maybe adding the warning when default settings are to be used as regards outputs and working directories.

All the stuff about absolute imports is OK.

File: fmriprep/run_workflow.py (hunk 2 above)
Comment (shoshber) [inline fmriprep/run_workflow.py]: reverted

**Pull request merged**